test/schema-serialization.test.js

```diff
@@ -60,6 +60,200 @@ test('custom serializer options', t => {
   })
 })
 
+test('Different content types', t => {
+  t.plan(32)
+
+  const fastify = Fastify()
+  fastify.addSchema({
+    $id: 'test',
+    type: 'object',
+    properties: {
+      name: { type: 'string' },
+      age: { type: 'number' },
+      verified: { type: 'boolean' }
+    }
+  })
+
+  fastify.get('/', {
+    schema: {
+      response: {
+        200: {
+          content: {
```

Comment:
Using `content` here does not ring nice to me. How about `contentTypes`?

Reply:
What do you mean? This is the short form?!

Reply:
Ah no worries, let's keep `content`.